src/types.ts

```diff
@@ -65,6 +71,7 @@ export interface IResolverOpts {
   ctx?: any;
 }
 
+/** The options that you can pass to `resolver.resolve(opts)` */
 export interface IResolveOpts extends IResolverOpts {
   // resolve a specific part of the source object
   jsonPointer?: string;
```

Comment:
Ah maybe I finally get it!

If I understand correctly, this property is telling "take the JSON object, go to this part specified here in particular and resolve THAT object, ignoring all the other keys around (which probably are just references that *can* be used in case they're used somewhere). Did I get it correctly this time?

Reply:
You got it :).